deploy/helm/sumologic/conf/logs/collector/otelcloudwatch/config.yaml

```diff
@@ -26,6 +26,29 @@ processors:
     send_batch_max_size: 2000
     send_batch_size: 1000
     timeout: 1s
+  transform/cloudwatch_logs:
+    error_mode: ignore
+    log_statements:
+      - context: resource
+        statements:
+          - set(attributes["k8s.pod.name"], attributes["cloudwatch.log.stream"])
+          - set(attributes["k8s.container.name"], attributes["cloudwatch.log.stream"])
+          - set(attributes["k8s.namespace.name"], attributes["cloudwatch.log.stream"])
+          - replace_pattern(attributes["k8s.pod.name"], "^.*kube\\.var\\.log\\.containers\\.([0-9a-zA-Z\\-]+)\\_.*", "$$1")
+          - replace_pattern(attributes["k8s.container.name"], "^.*kube\\.var\\.log\\.containers\\.([0-9a-zA-Z\\-]+)\\_.*", "$$1")
```

Comment:
Why is `k8s.pod.name` and `k8s.container.name` set to the same value? It looks like it's the pod name, which is fine in `k8s.pod.name`, but the `k8s.container.name` should rather contain the container name and not the pod name, right?

Reply:
Sample stream name: `from-fluent-bit-kube.var.log.containers.logger-server-67fc8884f4-4vwft_demo_main-3dad569bd5d74c70072b461f986b92da15c10d1b745b8686c65d1395a95bb361.log`
I don't remember well, but the container name is either `main` or `main-...`, so yes, the regex definitely should be modified

Reply:
right `logger-server-67fc8884f4-4vwft` is the pod name and `main` is the container name. I'll change the regex for the container name

Reply:
updated the container name regex in the PR